cache/ttlotel.go

```diff
@@ -0,0 +1,49 @@
+package cache
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+var tracer = otel.Tracer("cache")
```

Comment:
Any reason this is `otel.Tracer` and not just `otel.GetTracerProvider().Tracer(packageName)` like it conventionally is in Evergreen?

Reply:
No this is a mistake, I'll change it!